cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -831,6 +831,9 @@ private boolean addOrRemoveBroker(HttpServletRequest request, HttpServletRespons
       concurrentLeaderMovements = concurrentMovementsPerBroker(request, false);
       skipHardGoalCheck = skipHardGoalCheck(request);
       excludedTopics = excludedTopics(request);
+      if (brokerIds.isEmpty()) {
```

Comment:
It looks like whenever an endpoint supports `brokerIds`, it is a required parameter.
We would either (1) make this sanity check in `KafkaCruiseControlServletUtils#brokerIds(HttpServletRequest)`, or (2) ensure that all required sanity checks are performed after retrieval of brokerIds. Note that the same sanity check is missing in `#demoteBroker(HttpServletRequest, HttpServletResponse, EndPoint)`.

Reply:
Sorry if what I refer to `sanity check` here was not clear. I mean the following sanity check regarding whether `brokerIds` is `empty` or not.
```
if (brokerIds.isEmpty()) {
    throw new IllegalArgumentException("Target broker ID is not provided.");
}
```
Would it be possible to do ^this sanity check in (1) only in `KafkaCruiseControlServletUtils#brokerIds(HttpServletRequest)` or (2) both in `#demoteBroker(HttpServletRequest, HttpServletResponse, EndPoint)` and `addOrRemoveBroker(HttpServletRequest, HttpServletResponse, EndPoint)`?